docker-compose.yml

```diff
@@ -1,5 +1,14 @@
-version: "3.9"
 services:
+  postgres:
+    image: postgres:11.5-alpine
+    ports:
+      - "127.0.0.1:5436:5432"
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "postgres"]
+      interval: 1s
+    networks:
+      - default
+      - dbs
```

Comment:
This is the same as the [postgres service that's already in `docker-compose.yml`](https://github.com/hypothesis/report/blob/1acbfa2d64388541bac08e58ecf53db2cf6da5cc/docker-compose.yml#L40-L46), the cookiecutter just moves it to the top of the file